src/client/src/shell/openFin/openFin.ts

```diff
@@ -2,13 +2,16 @@ import { Observable } from 'rxjs'
 const LOG_NAME = 'OpenFin: '
 
 const REQUEST_LIMIT_CHECK_TOPIC = 'request-limit-check'
-
+const LIMIT_CHECKER_UUID = 'LIMIT-CHECKER'
+const LIMIT_CHECKER_STATUS_TOPIC = 'request-limit-check-status'
 export class OpenFinLimitChecker {
   private limitCheckSubscriber: string | null = null
   private limitCheckId: number = 1
 
   constructor() {
     if (typeof fin !== 'undefined') {
+      this.setLimitCheckSubscriber = this.setLimitCheckSubscriber.bind(this)
```

Comment:
Why don't we just use an arrow function instead of this-binding here? i.e.
```typescript
// Was: setLimitCheckSubscriber(uuid: string, topic: string) { ... }
setLimitCheckSubscriber = (uuid: string, topic: string) => { ... }
```
There are different opinions online about the 2 approaches but in RT codebase there seems to be a massive preference for the arrow function syntax (and it is less verbose). No showstopper but just for the sake of consistency of the code :)

Reply:
I tried that, and it did not work.